### ISSUE TITLE
[BUG] New bottles don't create anything beyond the folder

### ISSUE THREAD

**Issue by jcc10** [bottlesdevs/Bottles#1054]
**Describe the bug**
When creating a new bottle it only creates the bottle folder and (sometimes) the .yml file.

**Installation**
 - Method: Flatpak
 - Version: 2022.2.14-trento

**To Reproduce**
1. Create new bottle (any valid name)
2. If set to gaming will fail at dxvk stage (since the files don't exist)
3. If set to custom will silently fail
4. Open bottle and try to view drive contents, will fail due to dir not existing.

**Expected behavior**
It creates the bottle

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - OS: Pop!_OS / Ubuntu
 - Version: 21.10
 - Display server: Xorg

**Health Check**
```yml
Display:
    X.org: true
    X.org (port): :99.0
    Wayland: false
Graphics:
    vendors:
        nvidia:
            vendor: nvidia
            envs:
                __NV_PRIME_RENDER_OFFLOAD: '1'
                __GLX_VENDOR_LIBRARY_NAME: nvidia
                __VK_LAYER_NV_optimus: NVIDIA_only
            icd: /usr/lib/x86_64-linux-gnu/GL/vulkan/icd.d/nvidia_icd.json:/usr/lib/i386-linux-gnu/GL/vulkan/icd.d/nvidia_icd.json
    prime:
        integrated: null
        discrete: null
Kernel:
    Type: Linux
    Version: 5.15.11-76051511-generic
Distro:
    Name: GNOME
    Version: '"41 (Flatpak runtime)"'
Tools:
    cabextract: true
    p7zip: true
    patool: true
    glibc_min: '2.33'
Bottles_envs: null
```
**Additional context**
Let me know what error logs you want if any.

**Comment by jcc10** [comment]
I would like to add that it seems to constantly update the wine config in `wine: configuration in L"/home/digitalkitsune/.wine" has been updated.` instead of the bottle's wine directory so that may also be a issue.

**Comment by mirkobrombin** [comment]
There is something in your system which is forcing the WINEPREFIX env var to .wine. Like this one https://github.com/bottlesdevs/Bottles/issues/979#issuecomment-1030667845

**Comment by mirkobrombin** [comment]
Fixed in 2022.2.28
